src/live-share/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const VSLS_SHARE_COLOR_MEMENTO_NAME = 'peacock.vsls.share.color';
```

Comment:
All `Live Share` integration code goes to the `src/live-share` for isolation purposes.